src/collections/Posts/populateAuthors.ts

```diff
@@ -0,0 +1,27 @@
+import type { AfterReadHook } from "payload/dist/collections/config/types";
+
+export const populateAuthors: AfterReadHook = async ({
+  doc,
+  req: { payload },
+}) => {
+  if (doc?.authors) {
+    const authorDocs = await Promise.all(
+      doc.authors.map(
+        async (author) =>
```

Comment:
is there any issues where the author can be something other than an object? Wondering if there's a way we can ensure that it's always an object